internal/registry/json.go

```diff
@@ -27,12 +27,33 @@ const (
 	mediaTypeOCILayer    = "application/vnd.oci.image.layer.v1.tar+gzip"
 )
 
-// isMediaTypeImageLayerV1 returns true for "tar.gz" layer types referenced by imageManifestV1.LayerDigests.
+// isMediaTypeImageLayerV1 returns true for "tar.gz" layer types referenced by imageManifestV1 LayerDigests.
 // These are "Accept" headers for the OCI Registry "/v2/${Repository}/blobs/${Digest}" endpoint.
 func isMediaTypeImageLayerV1(mediaType string) bool {
 	return mediaType == mediaTypeOCILayer || mediaType == mediaTypeDockerLayer
 }
 
+// isMediaTypeImageConfigV1 returns true for a type represented by imageConfigV1
+func isMediaTypeImageConfigV1(mediaType string) bool {
+	return mediaType == "application/vnd.oci.image.config.v1+json" ||
+		mediaType == "application/vnd.docker.container.image.v1+json"
+}
+
+// The below are maps for convenience as we need both to enumerate the keys for "Accept" headers, and also check the
+// returned "Content-Type". These are internal and not exposed, so risk of map mutation is minimal.
```

Comment:
though the risk is minimal, I prefer having consistent experiences over checking predefined strings..

Reply:
by consistent experience, you want me to define a slice with two headers in it and then a function to iterate over the slice (* 2)?

Reply:
going for this!